device.go

```diff
@@ -228,6 +228,7 @@ func createDevice(devInfo DeviceInfo) Device {
 	} else if isBlock(path) {
 		return newBlockDevice(devInfo)
 	} else {
+		virtLog.Infof("Device %s has not been passed to the container", path)
```

Comment:
Could you make that "structured", so something like:

```
deviceLogger().WithField("device", path).Info("Device not passed to container")
```

Reply:
done